notes.txt

```diff
@@ -0,0 +1,8 @@
+This bot returns the top hit from google news related to the topic sent.
+
+searchArticle.js is from the repo
```

Comment:
searchArticle.js is licensed under MIT, which requires you to include it's license and copyright information. I would suggest creating a file along the lines of LICENSE_searchArticle.js.md and copying the contents of [this file](https://github.com/apollonian11/reactriot2017-reaction/blob/master/LICENSE) into it